frontend/src/features/permits/apiManager/permitsAPI.ts

```diff
@@ -116,17 +118,28 @@ const getApplications = async (
     orderBy = [],
   }: PaginationAndFilters,
   pendingPermitsOnly?: boolean,
+  applicationsInQueueOnly?: boolean,
 ): Promise<PaginatedResponse<ApplicationListItem>> => {
   const companyId = getDefaultRequiredVal("", getCompanyIdFromSession());
   const applicationsURL = new URL(APPLICATIONS_API_ROUTES.GET(companyId));
 
   // API pagination index starts at 1. Hence page + 1.
   applicationsURL.searchParams.set("page", `${page + 1}`);
   applicationsURL.searchParams.set("take", `${take}`);
-  applicationsURL.searchParams.set(
-    "pendingPermits",
-    `${Boolean(pendingPermitsOnly)}`,
-  );
+
+  if (pendingPermitsOnly !== undefined) {
```

Comment:
I think it might be better to do the check as ```if (typeof pendingPermitsOnly !== "undefined"```, similarly for applicationsInQueueOnly